impl/src/planning/subplanners.rs

```diff
@@ -759,3 +777,52 @@ impl<'planner> CrateSubplanner<'planner> {
     }
   }
 }
+
+
+#[cfg(test)]
+mod tests {
+  use crate::testing::make_workspace;
+  use crate::metadata::CargoMetadataFetcher;
+  use crate::planning::{BuildPlannerImpl, BuildPlanner};
+  use std::path::PathBuf;
+  use crate::settings::{GenMode, RazeSettings};
+  use std::collections::HashMap;
+
+  #[test]
+  fn sha256s() {
+    let settings = RazeSettings {
```

Comment:
There's a helper function for creating a `RazeSettings` object `settings::tests::dummy_raze_settings` that should be used here instead